modules/battenberg/1.0/config/default.yaml

```diff
@@ -0,0 +1,50 @@
+lcr-modules:
+    
+    battenberg:
+        inputs:
+            # Available wildcards: {seq_type} {genome_build} {sample_id}
+            sample_bam: "__UPDATE__"
+            battenberg_script: "{MODSDIR}/src/R/battenberg_wgs_hg38.R"
+            calc_sex_status: "{MODSDIR}/src/bash/calc_sex_status.sh"
+            cnv2igv: "{SCRIPTSDIR}/cnv2igv/1.0/cnv2igv.py"
+            #TODO: this should be tested with v1.2 of cnv2igv.py
+
+        scratch_subdirectories: []
+
+        reference_path:
+            hg38: "__UPDATE__"
+            grch37: "__UPDATE__"
+
+        options:
+            #update and add/remove these lines as needed for the reference genomes being used. 
+            chr_prefixed_reference:
+                hg38: " --chr_prefixed_genome "
+                grch37: " "
+            x_chrom: 
+                grch37: "X"
+                hg38: "chrX"
+            y_chrom: 
+                grch37: "Y" 
+                hg38: "chrY"
+
+        conda_envs:
+            battenberg: "{MODSDIR}/envs/battenberg_fresh.yml"
```

Comment:
I think with this, there are 2 extra yml files in this PR that I think are not refereed to, `battenberg_fresh.yaml` (seems to be in the parent directory) and `modules/battenberg/1.0/envs/battenberg-1.0.yaml`

Reply:
Can you comment directly on which ones should be deleted?